dms/models/base.py

```diff
@@ -17,3 +18,17 @@ def unlink(self):
             [("res_model", "=", self._name), ("res_id", "in", self.ids)]
         ).unlink()
         return result
+
+    @api.model
+    def search_panel_select_range(self, field_name):
```

Comment:
Why not overriding this in the specific models?

Reply:
And I prefer that you overwrite by monkey-patching the variable `SEARCH_PANEL_LIMIT` before calling super and restore after than verbatim copying code.

Reply:
Please read my second comment to see if feasible.

Reply:
Yes, it is feasible and much more optimal and readable.